numpy/core/src/umath/ufunc_object.c

```diff
@@ -87,6 +94,15 @@ _get_wrap_prepare_args(ufunc_full_args full_args) {
     }
 }
 
+static void
```

Comment:
Maybe move the functions up instead?

Reply:
I'd rather keep the diff smaller to make the blame easier

Reply:
Fine to make it another commit; I find these prototypes before the actual code very confusing...

Reply:
Done - I suppose in a standalone commit, a messy diff is fine if it comes with a commit message assuring nothing really changed.